src/app/workshops-list/workshops-list.component.ts

```diff
@@ -94,26 +94,26 @@ export class WorkshopsListComponent {
     }
     
     createUrl(page : number) : string {
-        let locations: string;
+        let location: number;
         let categories: string;
         let startDate: string;
         let endDate: string;
         let minPrice: string;
         let maxPrice: string;
         this.route.queryParams.subscribe(params => {
```

Comment:
i thought we decided to use simple snapshot ?

Reply:
snapshot doesn't work